cpp/vm/evmzero/interpreter.cc

```diff
@@ -966,6 +967,21 @@ struct Impl<OpCode::POP> {
   static OpResult Run() noexcept { return {}; }
 };
 
+template <>
+struct Impl<OpCode::PUSH0> {
+  constexpr static OpInfo kInfo = OpInfo{
+      .pops = 0,
+      .pushes = 1,
+      .static_gas = 2,
+      .introduced_in = EVMC_SHANGHAI,
+  };
+
+  static OpResult Run(uint256_t* top, Context&) noexcept {
```

Comment:
The context can be skipped.
```suggestion
  static OpResult Run(uint256_t* top) noexcept {
```